src/frontend/src/components/public/PIAFormTabs/review/index.tsx

```diff
@@ -246,12 +260,18 @@ const PIAReview = ({ printPreview }: IReviewProps) => {
                             className="d-flex align-items-center"
                             key={index}
                           >
-                            {Object(pia?.review?.programArea)?.reviews?.[role]
+                            {!allowUserReviewProgramArea() ||
```

Comment:
this logic is not very clear. Even if one of the program area reviewbyguid == to the current user guid it returns  false and they see the view mode? Seems convoluted.